### PR TITLE
Edit pass through our instructions

### PR DESCRIPTION
**Pull request by tswicegood**

### CODE REVIEW

**Comment by tswicegood** [comment]
@kmcq when we have Ruby required, we're making it harder for front end developers who might be better able to write something in Node to handle backend. I like using this as the base for the project, but letting folks deviate if they want to.

**Comment by kmcq** [comment]
cool thanks! that sounds good to me

File: README.md [hunk 1]
```diff
@@ -27,46 +27,58 @@ In the repository, the Rails team uses labels to define the components of the st
...
+- Building a fancy UI around a basic API of your own using other tools like JS.
+
+This is where you can _really_ show off who you are and what you can do. Just don't go too overboard on time.
+
+Make sure you:
 - Write clean, well tested, well organized code.
-- Leave comments if appropriate, but don't worry about commenting up everything, just explain any 'hard parts'.
-- Use the rails app in `./youearnedit` - this will help minimize changes so we can focus on your code.
+- Leave comments if appropriate. We don't comment everything. We just explain the "hard parts."
+
```
Comment (chrisbloom7) [inline README.md]: excess line?

File: README.md [hunk 2]
```diff
@@ -27,46 +27,58 @@ In the repository, the Rails team uses labels to define the components of the st
...
+- Use the rails app in `./youearnedit` as the base. Feel free to ignore parts that aren't relevant to your example. Using this as a starting point will help minimize changes so we can focus on your code.
+
```
Comment (chrisbloom7) [inline README.md]: excess line?

**Pull request merged**